File: freecodecampjs/slasher.js
// INPUT: Obtain an array and a number
// STEPS:
// Start from the beginning of the array and remove n amount from the beg. of the array
// Return slashed array
// OUTPUT: Returns the array with the n amount removed from the beg. of the array

function slasher(arr, howMany) {
    for(var i = 0; i < howMany; i++) {
          arr.shift();
    }
  return arr;
};

console.log(slasher([1, 2, 3], 2));
console.log(slasher([1, 2, 3], 0));
console.log(slasher([1, 2, 3], 9));
console.log(slasher([1, 2, 3], 4));

File: freecodecampruby/slasher.rb
#INPUT: Obtain an array and a number
#STEPS:
#Delete n from the beginning of the array
#Slice the array based on the number parameter
#OUTPUT: Return sliced array
def slasher(array, how_many)
  p array.drop(how_many)
end

slasher([1, 2, 3], 2)
slasher([1, 2, 3], 0)
slasher([1, 2, 3], 9)
slasher([1, 2, 3], 4)